scripts/text_to_speech_node.py

```diff
@@ -15,8 +15,11 @@
 import re
 import sys
 
-reload(sys)
-sys.setdefaultencoding('utf8')
+try:
```

Comment:
Why do you let an exception silently pass? Maybe add a comment explaining it before someone removes it considering it bad practice.

Reply:
As it is python2/python3. But I can add that. That explains a lot.